fix(curriculum): Provide Example for Media Query (#47422)
freeCodeCamp/freeCodeCamp: curriculum/challenges/english/14-responsive-web-design-22/learn-responsive-web-design-by-building-a-piano/612ec19d5268da7074941f84.md

---

# --description--
In order to query 2 media condition, the `and` keyword is used.

> For example, a media query that targets a display width between 500px and 1000px would be:

Reviewer: to fix the eslinit issue, you can add space between the quotes and this line

```suggestion
For example, a media query that targets a display width between 500px and 1000px would be:

```

---

# --description--

> In order to query two media conditions, the `and` logical operator is used. This one of many used to construct media queries.

Reviewer 2: I think this is a bit more straightforward language:
```suggestion
Logical operators can be used to construct more complex media queries. The `and` logical operator is used to query two media conditions.
```

Reviewer: I have edited Jeremy comment for the needed format, and to avoid spamming the pr.

> In order to query two media conditions, the `and` logical operator is used. This one of many [logical operators](https://www.freecodecamp.org/news/media-query-css-example-max-and-min-screen-width-for-mobile-responsive-design/) used to construct media queries.

Reviewer 3: The challenge should not contain outbound links.

> [Challenge descriptions/instructions](https://contribute.freecodecamp.org/#/how-to-work-on-coding-challenges?id=challenge-descriptionsinstructions)
> 
> Don't use outbound links. These interrupt the flow. Campers should never have to google anything during these challenges.

Author: Ill remove the link and convert to PR.
